great-idea-website/css/index.css

```diff
@@ -63,4 +63,105 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+* {
+	box-sizing: border-box;
+	color: black;
```

Comment:
Personally i would advise against setting `color` on universal selector. Body or container would be better place.